README.md

```diff
@@ -56,6 +56,16 @@ mount-point = /var/tmp
 
 This will set up a /dev/zram1 with ext2 and generate a mount unit for /var/tmp.
 
+In case you want this path to be user-writable, you can use following
+"high-quality hack" until `systemd-makefs` provides a proper mechanism to
+set ownership of a generated filesystem. For the above example, create a
+`/etc/systemd/system/systemd-zram-setup@zram1.service.d/override.conf` file:
+
+```ini
+[Service]
+ExecStartPost=/bin/sh -c 'd=$(mktemp -d); mount "$1" "$d"; chmod 777 "$d"; umount "$d"; rmdir "$d"' _ /dev/%i
```

Comment:
/var/tmp (and /tmp) is *not* 777 (indeed, if you have anything public 777, that's a Problem).

They're 1777 – a=rwx,o+t.

Reply:
Very good point, thanks. Had to refresh my sense of what that sticky bit is about..